### PR TITLE
[enterprise-4.2] Remove new docs on configuring custom PKI using Proxy API

### PR DESCRIPTION
**Pull request by codyhoag**
Not yet backported to 4.2 (4.2 progress tracked [here](https://issues.redhat.com/browse/MON-884)). Only available in 4.3 (which is already published).

### CODE REVIEW

**Comment by codyhoag** [comment]
@danehans can you give a final thumbs up that this should be removed from 4.2 docs until it's backported?

Preview for setting custom default certificate: http://file.rdu.redhat.com/~choag/02032020/custom-pki-revert/networking/ingress-operator.html#nw-ingress-setting-a-custom-default-certificate_configuring-ingress

(Configuring a custom PKI assembly is removed.)

File: modules/nw-ingress-setting-a-custom-default-certificate.adoc [hunk 1]
```diff
@@ -25,6 +23,13 @@ NAME      AGE
 default   10m
 ----
 
+[WARNING]
```
Comment (kalexand-rh) [inline modules/nw-ingress-setting-a-custom-default-certificate.adoc]: Is this warning an elaboration of the first prereq? If so, I'd make it active and attach it to that prereq.

(Will the user know how to make sure the CA bundle is provided by the container userspace? (Also, userspace is only used once in the collection. Is there another way to convey this idea?)

**Review by kalexand-rh — COMMENTED** [review]
Two questions, and it LGTM

**Comment by codyhoag** [comment]
This has been approved via email. The ability to privately sign certs was confirmed in 4.2.x, but it only worked in certain cases. The [official backport](https://issues.redhat.com/browse/MON-884) is still in progress; we'll add this content back when the backport has finished.

**Pull request merged**